src/cli/README.md

```diff
@@ -2047,6 +2047,20 @@ Set the number of indirect TX retries on the MAC layer.
 Done
 ```
 
+### mac send \<op\>
```

Comment:
nit: can we merge this command with `fake` command which is currently used to send fake `/a/an` for certification purpose.  
e.g. 
```bash
> fake datarequest
Done
> fake emptydata
Done
```
@librasungirl @Irving-cl

Reply:
No strong preference on this. I think we can keep it as it is now since `fake` command sends a IP message and needs a IP address while `mac send` sends a Link layer message and always sends to the parent. But we can consider again in the future if we need a specific command to send these kind of messages when there's more of them.